### PR TITLE
Update readme to show how to build and run hello world

### PR DESCRIPTION
**Pull request by cnmade**
Add new basic hello world example.
Update readme to show how to build and run with the basic example
Fixed normal missing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,19 @@ C++ client for [ClickHouse](https://clickhouse.com/).
 * Int128
 * UUID
 
+## Requirement
+
+You need gcc cmake libabsl-dev to build this project.
```
Comment (filimonov) [inline README.md:27]: 1) Should work without `libabsl-dev`
2) gcc version 7 at least (clang >= 6 also supperted, after #131 MSVC should work also).

Comment (filimonov) [reply README.md:27]: To build with openssl libssl-dev is also needed.

Comment (filimonov) [reply README.md:27]: See https://github.com/ClickHouse/clickhouse-cpp/blob/master/.github/workflows/linux.yml

File: tests/helloworld/main.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    /// Delete table.
+    client.Execute("DROP TEMPORARY TABLE test_client");
+}
+
+
+static void RunTests(Client& client) {
+    GenericExample(client);
+}
+
+int main() {
```
Comment (filimonov) [inline tests/helloworld/main.cpp:73]: Vs https://github.com/ClickHouse/clickhouse-cpp/tree/master/tests/simple ?

Comment (cnmade) [reply tests/helloworld/main.cpp:73]: the simple directory source code not that simple, you can see so many code blocks.
Shall we start from very basic hello world example?

File: README.md (hunk 1 above)
Comment (cnmade) [reply README.md:27]: Hello world does not need https i think. or am i wrong?

I just finish running the hello world for me ,so it's time to share how to run example code , may help who want to play .

**Comment by cnmade** [comment]
If this is not ok for any inconvence, i will close this pull request. 
For me, i know how to play , job finished.

File: tests/helloworld/main.cpp (hunk 2 above)
Comment (filimonov) [reply tests/helloworld/main.cpp:73]: OK for me. Just want to understand your motivation.

File: README.md (hunk 1 above)
Comment (filimonov) [reply README.md:27]: > Hello world does not need https i think

that readme is for the clickhouse-cpp, why not make it clear for all the cases?

**Comment by filimonov** [comment]
It's almost ok. 

Your emotional reaction is not entirely clear. My comments were not correct / seemed rude?

Comments/minor remarks are a natural element of code reviews.

If you trying to make things better why do you stop half-road?

**Comment by cnmade** [comment]
@filimonov I am think is it my pull request too lower level, did really other people need it ?  It's not about your comment/review.
I actually agree your point. and i will always reconsider myself.

**Pull request closed without merging**